pages/architecture/certificates.rst

```diff
@@ -12,12 +12,41 @@ This does not give the foundation super powers since vendors and users (both val
 .. raw:: html
     :file: ../../_static/images/certificates.svg
 
-Extensions
-**********
+Vendor CA Certificate
+*********************
+
+Common Name: CN=...,O=...,C=NL
```

Comment:
CN is no longer used, altNames is the way to go

Reply:
Then I'll add it as an extra subjectAltName entry, since it should have a human-readable component which also specifies which certificate it is (CA/TLS CA/etc).

Reply:
better describe it as optional, maybe vendors want to use it for internal administration